src/app/components/forms/user/password-reset/password-reset.component.ts

```diff
@@ -0,0 +1,88 @@
+import { Component, signal, inject } from '@angular/core';
+import { FormControl, FormGroup, Validators, ReactiveFormsModule } from '@angular/forms';
+import { MatFormFieldModule } from '@angular/material/form-field';
+import { MatInputModule } from '@angular/material/input';
+import { MatButtonModule } from '@angular/material/button';
+import { CommonModule } from '@angular/common';
+import { TranslatePipe } from '@ngx-translate/core';
+import { I18nService } from '@services/general';
+import { ChangePwHttpService } from 'app/services/http/change-pw-http-service/change-pw-http.service';
+import { AuthHttpService } from '@services/http';
+
+/**
+ * @author @aBlancoC
+ * 
+ * Component for handling user password reset functionality.
+ * 
+ * This Angular standalone component provides a form for users to change their password.
+ * It includes validation for required fields and minimum password length, and ensures
+ * that the new password and confirmation match before submitting the change request.
+ * 
+ * Dependencies:
+ * - I18nService: For internationalization support.
+ * - ChangePwHttpService: Service to handle password change HTTP requests.
+ * - AuthHttpService: Service to access current user authentication information.
+ * 
+ * Features:
+ * - Reactive form with validation for old password, new password, and confirmation.
+ * - Loading and submission state management.
+ * - Password match validation.
+ * - Handles form submission and invokes password change service.
+ */
+@Component({
+  selector: 'app-password-reset',
+  standalone: true,
+  imports: [
+    CommonModule,
+    ReactiveFormsModule,
+    MatFormFieldModule,
+    MatInputModule,
+    MatButtonModule,
+    TranslatePipe
+  ],
+  templateUrl: './password-reset.component.html',
+  styleUrl: './password-reset.component.scss'
+})
+export class PasswordResetComponent {
+  readonly i18nService = inject(I18nService);
+  private readonly ChangePasswordService = inject(ChangePwHttpService);
+  //const user = authHttpService.currentUser();
+  private readonly AuthHttpService = inject(AuthHttpService);
```

Comment:
siguiendo el estandar: quedaria ah si


```suggestion
  private readonly authHttpService = inject(AuthHttpService);
```